src/index.mjs

```diff
@@ -27,78 +35,162 @@ function directoryHasPackageJson(dir) {
 	return fs.readdirSync(dir).some((file) => file === 'package.json');
 }
 
+/**
+ * @param {string} pkgManager
+ * @return {string} The command
+ */
+function getExecuteCommand(pkgManager) {
+	switch (pkgManager) {
+		case 'npm':
+			return 'npx';
+		case 'bun':
+			return 'bunx';
+		case 'pnpm':
+		case 'yarn':
+			return `${pkgManager} dlx`;
+		default:
+			throw new Error(`"${pkgManager}" package manager is not supported`);
+	}
+}
+
+/**
+ * @param {string} errorMessage
+ */
+function cancelSetup(errorMessage = '') {
+	console.error(
+		stripIndent(`
+			${picocolors.red(picocolors.bold('Setup canceled!'))}
+		`),
+	);
+
+	if (errorMessage) {
+		console.error(
+			stripIndent(`${errorMessage}
+			`),
+		);
+	}
+
+	process.exit(1);
+}
+
 /**
  * @param {string} pkgManager
  */
-function getInstallCommand(pkgManager) {
-	return pkgManager === 'npm' ? 'install' : 'add';
+async function showPrompt(pkgManager) {
+	console.info(
+		stripIndent(`
+			We'll create a '${DEFAULT_CONFIG_FILE}' file containing:
```

Comment:
[suggestion] How about emphasizing the file name with a color like this?

```suggestion
			We'll create a ${picocolors.cyan(DEFAULT_CONFIG_FILE)} file containing:
```

Reply:
I've used italics so we can keep a simple semantic colour palette, i.e. red for errors, green for success and blue for links